### DISCUSSION TITLE
idf.py menuconfig  - did not find file Compiler/-asm

### DISCUSSION THREAD

**Discussion by soky** [Mair/esp32-course#148] [Course Qeustions]
I already have ESP-IDF installed for the Learn ESP32 course. When trying to run idf.py menuconfig in esp-idf command prompt on in Powershell i get the error...did not find file Compiler/-asm.
What should I do?  Should I install esp-idf for Windows as well?

**Comment by Mair** [comment]
to run that command you need to be in a directory of a project. The IDF environment needs to be loaded and also make sure your directory has no spaces.
What OS are you running?

**Reply by soky** [reply]
Hi Mair Im running Win 11.

**Reply by Mair** [reply]
so what do you mean when you say 
> Should I install esp-idf for Windows as well?
